lib/presentation/pages/plan_expenses_page/plan_expenses_page.dart

```diff
@@ -0,0 +1,164 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:trip_tally/presentation/utils/enums/context_extensions.dart';
+import 'package:trip_tally/presentation/widgets/icon_button_text_field.dart';
+
+import '../../theme/app_dimensions.dart';
+import '../../theme/app_paths.dart';
+import '../../utils/router/app_router.dart';
+import '../../widgets/app_scaffold.dart';
+import '../../widgets/circle_svg_button.dart';
+import '../../widgets/custom_elevated_button.dart';
+import '../../widgets/expanses_icon_list.dart';
+import '../../widgets/expense_icon_contaner.dart';
+import '../../widgets/main_container.dart';
+import '../../widgets/suffix_icon_text_field.dart';
+
+@RoutePage()
+class PlanExpensesPage extends StatelessWidget {
+  const PlanExpensesPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return AppScaffold(
+      actions: [
+        CustomSvgButton(
+          svgPath: AppPaths.arrowBack,
+          onTap: () => context.router.push(const HomeRoute()),
+        ),
+        const Spacer(),
+        CustomSvgButton(
+          svgPath: AppPaths.calendar,
+          onTap: () {},
+        ),
+        CustomSvgButton(
+          svgPath: AppPaths.package,
+          onTap: () {},
+        ),
+        CustomSvgButton(
+          svgPath: AppPaths.person,
+          onTap: () {},
+        ),
```

Comment:
you could make this actions as a button widgets to have the on tap action in every page on the app